modules/GanneffServ.rb

```diff
@@ -574,8 +639,13 @@ def akill(client, reason, operreason, channel="")
       ret = kill_user(client, reason)
     else # if host
       reason = "#{reason}|#{operreason}"
-      debug(LOG_DEBUG, "Issuing AKILL: *@#{host}, #{reason} lasting for #{@akill_duration} seconds")
-      ret = akill_add("*@#{host}", reason, @akill_duration)
+      if client.host =~ /#{@protected_patterns}/i # if protected hosts
+        debug(LOG_DEBUG, "Using /kill instead of AKILL for protected user #{client.name}")
+        ret = kill_user(client, reason)
```

Comment:
This leaks the oper reason; kills don't hide oper reasons like akills do; easy fix is to just put this section as a branch in the enclosing if